packages/fhir-converter/src/templates/cda/Utils/GeneratePractitionerId.hbs

```diff
@@ -6,9 +6,6 @@
             {{else if FirstId.root}}
                 {{!--If only root is provided, it is supposed to be unique--}}
                 "{{generateUUID (toString FirstId.root)}}"
-            {{else}}
```

Comment:
when there is no extension (NPI code) or root (NPI), we just generate a uuid based off of the json object which has literally 0 connection to any existing practioner ID